api/MediaMetadata.json

```diff
@@ -29,7 +29,7 @@
             }
           ],
           "firefox_android": {
-            "version_added": null
+            "version_added": "82"
```

Comment:
This is part of the `MediaSession` API, but that's not updated in this PR.

Per https://bugzilla.mozilla.org/show_bug.cgi?id=1665496 this was supported on Android, and from there I found my way to https://github.com/mdn/browser-compat-data/pull/6845 by searching for @hamishwillee's PRs.

This change would conflict, so I suggest backing it out.